docs/standard/native-interop/interop-guidance.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Native interoperability best practices
+description: Learn the best practices for interfacing with native components in .NET.
+author: jkoritzinsky
+ms.author: jekoritz
+ms.date: 01/18/2019
+---
+
+# Native interoperability best practices
+
+## General guidance
```

Comment:
Right after the H1 `#` title you should have an introductory paragraph that lets the reader know what this article is about. This info is also capture in search and helps people find your article. You should never start an H2 `##` alone below an H1.